deepcell/metrics.py

```diff
@@ -157,14 +159,14 @@ class ObjectAccuracy(object):  # pylint: disable=useless-object-inheritance
     Warning:
         Position indicies are not currently collected appropriately
     """
-    # TODO: Implement recording of object indices for each error group
     def __init__(self,
                  y_true,
                  y_pred,
                  cutoff1=0.4,
                  cutoff2=0.1,
                  test=False,
-                 seg=False):
+                 seg=False,
+                 force_event_links=False):
```

Comment:
Update the docstring to include documentation of this parameter. I think it would also be appropriate in the general description of the function to describe when a user would want to use `force_event_links`.

Reply:
Got it, let me know if that looks like enough detail or if I should add more